examples/css/CHANGELOG.md

```diff
@@ -4,7 +4,7 @@
 
 ### Patch Changes
 
-- d5e1943: set filename to '[name].[platform].bundle' to eliminate difference between different versions.
+- 1022be8: Update filter demo.
 
```

Comment:
You don't need to manually edit the `CHANGELOG.md`

```suggestion
```

And you also change the wrong `CHANGELOG.md` :)